### ISSUE TITLE
Converting Level.sav file generates exceptions

### ISSUE THREAD

**Issue by Riscon** [wonjoonSeol-WS/Palworld-fix-uuid#1]
Hello,

thank you very much for your work.
I think the script no longer works for the current version - **v.0.2.3.0** as of 23/04/2024 (Day/Month/Year).
I get exceptions when editing the Level.sav file.

The exception log is very long, so I'm only posting the last part of it.

![image](https://github.com/wonjoonSeol-WS/Palworld-fix-uuid/assets/7253901/ae255b23-64e1-42ba-ab30-a1bdec9a4351)

**Comment by wonjoonSeol-WS** [comment]
Apologies, I must have slipped your comment in my inbox. 
Seems like newer version of the dependency package had been updated.

And the older version of the save tool no longer works for the recent Palworld save files. 
let me update the script. Thanks for the report

**Comment by wonjoonSeol-WS** [comment]
```
Publishing palworld-fix-uuid (0.1.2) to PyPI
 - Uploading palworld_fix_uuid-0.1.2-py3-none-any.whl 100%
 - Uploading palworld_fix_uuid-0.1.2.tar.gz 100%
```

0.1.2 version is now live, using the latest save tools dependency (0.23.0). Thanks!
